### DISCUSSION TITLE
help Notification Images error！

### DISCUSSION THREAD

**Discussion by Chenjiujiu** [invertase/react-native-firebase#7104] [Q&A] [answered]
I found an error when I used Notification Images

<img width="1564" alt="图片" src="https://user-images.githubusercontent.com/28746442/236489405-663c380e-4c84-4aad-bb2b-359f468d6988.png">

<img width="660" alt="图片" src="https://user-images.githubusercontent.com/28746442/236490102-db9d9bb8-4d71-47ad-afa7-fc6602d714d6.png">
<img width="611" alt="图片" src="https://user-images.githubusercontent.com/28746442/236490133-8768e6f6-f5c2-458f-b649-0e129e5ec695.png">

**Comment by khasbilegt** [comment, marked as answer]
Try setting the `use_frameworks! :linkage => :static` above the `pod 'Firebase/Messaging', '~> 10.7.0'`
